src/main/java/org/rxbooter/injector/core/Node.java

```diff
@@ -22,7 +22,7 @@
 
 import org.rxbooter.injector.Injector;
 import org.rxbooter.injector.Key;
-
+//guys seem to be in love with lombock so maybe you wanna do it ;P
 /**
```

Comment:
I'm trying to keep dependencies as minimal as possible. Lombok is not the one which is strictly necessary here.